streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -293,7 +294,18 @@ public void commitOffsets() {
                 consumedOffsetsAndMetadata.put(partition, new OffsetAndMetadata(offset));
                 stateMgr.putOffsetLimit(partition, offset);
             }
-            consumer.commitSync(consumedOffsetsAndMetadata);
+            try {
+                consumer.commitSync(consumedOffsetsAndMetadata);
+            } catch (final CommitFailedException cfe) {
+                for (Map.Entry<TopicPartition, OffsetAndMetadata> topicPartitionOffsetAndMetadataEntry : consumedOffsetsAndMetadata.entrySet()) {
```

Comment:
Yes. It's not severely important, but it's the code style we try to introduce throughout the whole Streams code base: `final` everywhere if possible.